lib/api/core/plugins/pluginContext.js

```diff
@@ -96,7 +115,7 @@ class PluginContext {
         get: () => {
           return {
             bootstrap: pluginInternalEngine.bootstrap.all.bind(pluginInternalEngine.bootstrap),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 95 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)